src/vertex_enumeration.jl

```diff
@@ -0,0 +1,206 @@
+function nonnegativeorthant_hrep(dim::Int)
+    h = Vector{HalfSpace{Float64, Vector{Float64}}}()
+    for i in 1:dim
+        e_i = zeros(dim)
+        e_i[i] = -1.
+        push!(h, HalfSpace(e_i, 0.))
+    end
+    H = h[1]
+    for i in 2:lastindex(h)
+        H = H ∩ h[i]
+    end
+    H
+end
+nonnegativeorthant(dim::Int) = polyhedron(nonnegative_hrep(dim))
+
+
+function br_envelope_hrep(player::Player)
+    A = player.payoff_array
+    h = Vector{HalfSpace{Float64, Vector{Float64}}}()
+    for i in 1:num_actions(player)
+        A_i = A[i,:]
+        push!(h, HalfSpace(A_i, 1.))
+    end
+    H = h[1]
+    for i in 2:lastindex(h)
+        H = H ∩ h[i]
+    end
+    H
+end
+
+
+function bestresponsepolyhedra(g::NormalFormGame)
+    nnorthantP = nonnegativeorthant_hrep(num_actions(g.players[1]))  # x_i ≥ 0 for all i=1,…,m
+    nnorthantQ = nonnegativeorthant_hrep(num_actions(g.players[2]))  # x_i ≥ 0 for all i=m+1,…,m+n
+    brenvelopeP = br_envelope_hrep(g.players[2])  # B'x ≤ 1
+    brenvelopeQ = br_envelope_hrep(g.players[1])  # Ax ≤ 1
+    P = brenvelopeP ∩ nnorthantP  # best response polyhedron for Player 1
+    Q = nnorthantQ ∩ brenvelopeQ  # best response polyhedron for Player 2
+    polyhedron(P), polyhedron(Q)
+end
+
+
+function hlabels(P::HRepresentation)
+    Phindices = Polyhedra.Index{Float64, HalfSpace{Float64, Vector{Float64}}}[]
+    for pi in eachindex(halfspaces(P))
+        push!(Phindices, pi)
+    end
+    Phindices
+end
+hlabels(P::DefaultPolyhedron) = hlabels(hrep(P))
+hlabels(P::VRepresentation) = hlabels(doubledescription(P))
+
+
+#function vlabels(P::VRepresentation)
+#    Pvindices = Polyhedra.Index{Float64, Vector{Float64}}[]
+#    for pi in eachindex(points(P))
+#        push!(Pvindices, pi)
+#    end
+#    Pvindices
+#end
+# vlabels(P::DefaultPolyhedron) = vlabels(vrep(P))
+# vlabels(P::HRepresentation) = vlabels(doubledescription(P))
+
+
+label_to_integer(idx::Polyhedra.Index{T, S}) where {T, S} = idx.value
+# integer_to_vlabel(n::Int) = Polyhedra.Index{Float64, Vector{Float64}}(n)
+# integer_to_hlabel(n::Int) = Polyhedra.Index{Float64, HalfSpace{Float64, Vector{Float64}}}(n)
+
+
+function labelmap(P::DefaultPolyhedron)
+    labelmaps = []
+    vpoints = [x for x in points(P)]
+    for pi in eachindex(points(P))
+        push!(labelmaps, (vpoints[label_to_integer(pi)], incidenthalfspaceindices(P, pi)))
+    end
+    Dict(labelmaps)
+end
+
+
+mutable struct LabeledPolyhedron{S<:Polyhedron, T<:Vector, U<:Vector, V<:Dict}
+    polyhedron::S
+    points::T
+    hlabels::U
+    labelmap::V
+end
+
+
+function LabeledPolyhedron(P::DefaultPolyhedron)
+    vpoints = [x for x in points(P)]
+    LabeledPolyhedron(P, vpoints, hlabels(P), labelmap(P))
+end
+
+
+struct LabeledBimatrixGame
+    game::NormalFormGame
+    P::LabeledPolyhedron
+    Q::LabeledPolyhedron
+end
+
+
+function LabeledBimatrixGame(g::NormalFormGame)
+    @assert num_players(g) == 2

```

Comment:
You can restrict the input type to `NormalFormGame` with 2 players:

```suggestion
function LabeledBimatrixGame(g::NormalFormGame{2})
```

Then you don't need `@assert num_players(g) == 2`.